src/tools/rbd/Utils.cc

```diff
@@ -389,6 +501,81 @@ int get_pool_image_snapshot_names(const po::variables_map &vm,
   return 0;
 }
 
+int get_pool_image_with_id_snapshot_names(const po::variables_map &vm,
```

Comment:
Same comment here -- positional "[pool/]image-name@snap" format shouldn't be possible